apps/hashdotai/blog/000_what-is-agent-based-modeling.mdx

```diff
@@ -0,0 +1,55 @@
+While traditional data science involves the analysis of historical datasets and existing information, [agent-based modeling](https://hash.ai/glossary/agent-based-modeling), also known as ABM, takes a quite different approach.
+
+ABM is inherently forward looking, and does not require large volumes of high-quality historical data: only a rough idea of how a system or its component parts might work. Starting from this conception, a microscale digital ‘model’ is built that attempts to approximate the results which might be expected in the real-world.
+
+## What is a model?
+
+Models attempt to represent and simulate real or imagined scenarios, which we call the ‘target’.
+
+Models are great because they force us to consider the many features of a target, and express the relationships between them. This in turn [allows for discovery](<http://onlinelibrary.wiley.com/doi/10.1002/(SICI)1520-6858(1999)2:2%3C69::AID-SYS3%3E3.0.CO;2-0/abstract>) of potential emergent phenomenon.
```

Comment:
```suggestion
Models are great because they force us to consider the many features of a target, and express the relationships between them. This in turn [allows for discovery](<http://onlinelibrary.wiley.com/doi/10.1002/(SICI)1520-6858(1999)2:2%3C69::AID-SYS3%3E3.0.CO;2-0/abstract>) of potential emergent phenomena.
```